setup.py

```diff
@@ -66,16 +66,17 @@ def write_version_py():
 __version__ = '{}'
 short_version = '{}'
 version_info = ({})
+version_hash = "{}"
```

Comment:
We do not need to write the version file dynamically. Version number can be hard coded and the git hash can be computed on the fly.